cli/cmd/root.go

```diff
@@ -82,9 +82,17 @@ var rootCmd = &cobra.Command{
 			}
 		}
 
+		if err = CheckLockedHappyVersion(cmd); err != nil {
+			return err
+		}
```

Comment:
```suggestion
		err = CheckLockedHappyVersion(cmd)
		if err != nil {
			return err
		}
```
I think this is more conventional for this repo